vnext/Microsoft.ReactNative/XamlUIService.idl

```diff
@@ -24,7 +24,10 @@ namespace Microsoft.ReactNative {
 
     // This needs to be manually provided to the ReactInstanceSettings when using XamlIslands
     static void SetXamlRoot(IReactPropertyBag properties, XAML_NAMESPACE.XamlRoot xamlRoot);
+    // This needs to be manually provided to the ReactInstanceSettings when using XamlIslands
+    static void SetAccessibleXamlRoot(IReactPropertyBag properties, XAML_NAMESPACE.XamlRoot xamlRoot);
```

Comment:
I think we want to change this to something like
`static void SetAccessibleRoot(IReactPropertyBag properties, XAML_NAMESPACE.FrameworkElement accessibleFrameworkElement)`

(and related changes to match)

Reply:
please also add documentation for this new API by annotating it with DOC_STRING("...");
As long as you're here, adding a doc for Get/Set XamlRoot would be good too :)

Reply:
Got it! Will do!